website/docs/r/workspaces_directory.html.markdown

```diff
@@ -10,9 +10,37 @@ description: |-
 
 Provides a WorkSpaces directory in AWS WorkSpaces Service.
 
+~> **NOTE:** AWS WorkSpaces service requires [`workspaces_DefaultRole`](https://docs.aws.amazon.com/workspaces/latest/adminguide/workspaces-access-control.html#create-default-role) IAM role to operate normally.
+
 ## Example Usage
 
 ```hcl
+data "aws_iam_policy_document" "workspaces" {
```

Comment:
I'd prefer to see the resource that's being documented at the top. Otherwise, it can get buried in the details

Reply:
I don't mind. Does the reverse order make sense?

Reply:
Reverse, or even just move the `aws_workspaces_directory` to the top